core/client/app/components/gh-user-invited.js

```diff
@@ -16,8 +17,10 @@ export default Ember.Component.extend({
     actions: {
         resend: function () {
             var user = this.get('user'),
-                notifications = this.get('notifications');
+                notifications = this.get('notifications'),
+                self = this;
```

Comment:
There's been quite a few places where I've been missing arrow functions. I remember some discussion about certain es6 features and our tooling not coping with it too well although I'm not sure if that was code climate, linting, or something else.

Would definitely be good to open a discussion on fat arrows and deciding whether we're standardizing on `export default Ember.Thing.extend({})` or `var Thing = {}; export default Thing;`.